### PR TITLE
List how content changes during common pipeline steps

### PR DESCRIPTION
**Pull request by philschatz**

### CODE REVIEW

File: docs-pipeline.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
+# How content changes in enki
+
+This describes the overall steps that content goes through.
+Several of these can be clumped together instead of being separate and would reduce the amount of validation & debugging/testing.
```
Comment (Stackblocks) [inline docs-pipeline.md:4]: This line needs some polish or a rework.
- What are "these?"
- Are they clumped together in the document to form a conceptual block? 
- Are they clumped together process-wise? 
- Are they currently clumped together at all or is it a suggestion that they could be clumped together (and if so is a reference doc the right place for that?)

File: docs-pipeline.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+This just runs an authenticated `git clone` and checks out the correct branch/commit.
+
+**Validation:**
+
+Use POET CLI to validate the results.
+
+
+## git-fetch-metadata
+
+3 things happen.
```
Comment (Stackblocks) [inline docs-pipeline.md:47]: "3" here should probably be written out to avoid parsing it as an out-of-place `ol` item

**Review by Stackblocks — CHANGES_REQUESTED** [review]
I've requested some changes for readability, but overall I really like this. 

It clears up a lot of the fog of mystery around the individual steps, and good clear documentation is always a plus.

**Review by ckline-tryptic — APPROVED** [review]

File: docs-pipeline.md (hunk 1 above)
Comment (philschatz) [reply docs-pipeline.md]: Thanks! I clarified the idea a little bit

**Review by Stackblocks — APPROVED** [review]

**Pull request merged**